app/views/users/edit_pass.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Update My Password</h1>
+<%= form_tag '/password', method: :post do %>
+  <%= hidden_field_tag :name, "#{@user.name}" %>
+  <%= hidden_field_tag :address, "#{@user.address}" %>
+  <%= hidden_field_tag :city, "#{@user.city}" %>
+  <%= hidden_field_tag :state, "#{@user.state }"%>
+  <%= hidden_field_tag :zip, "#{@user.zip}" %>
+  <%= hidden_field_tag :email, "#{@user.email}" %>
+  <%= label_tag :password %>
+  <%= text_field_tag :password %>
```

Comment:
I think we want to use a special tag that conceals the password. I forget what its called but it was in the lesson!

Reply:
Found it, password_field_tag, thanks for the catch!